components/cmp/ui/src/CmpUi/index.js

```diff
@@ -31,12 +31,11 @@ export default function CmpUiContainer({
 
   const _handleAccept = async evt => {
     evt && evt.preventDefault()
-
-    const {
-      purposeConsents,
-      vendorConsents
-    } = await getPurposesAndVendors.execute()
-    await sendConsents.execute({purposeConsents, vendorConsents})
+    try {
```

Comment:
You need to send the consents you're getting still. The line 39 is missing and is needed.